README.md

```diff
@@ -290,7 +290,7 @@ If you fork the project on GitHub, you can run your fork's [build workflow](.git
 <!-- MANPAGE: BEGIN EXCLUDED SECTION -->
     yt-dlp [OPTIONS] [--] URL [URL...]
 
-`Ctrl+F` is your friend :D
+`Ctrl+F`/`Command-F` is your friend :D
```

Comment:
what about something like this
```suggestion
Tip: Use `CTRL`+`F` (or `Command`+`F`)  to search by keywords
```

Reply:
Seems over and above the minimal change required to fix the text, but sure, if that's what the maintainers prefer. 🤷‍♂️